### DISCUSSION TITLE
What types of metadata is key?

### DISCUSSION THREAD

**Discussion by wonderchook** [opensupplyhub/supplychaindata.exchange#20] [General]
In exchanging data there are use cases where metadata is important. In other use cases who you are exchanging with you might trust sufficiently to not need individual record metadata. 

What types of metadata is needed?

Geocoordinates: how those coordinates were created
Number of workers: where did that information come from

What else?

**Comment by vasgat** [comment]
I think there is a need to also capture the status of an organization or location or have an expiration date on the data, especially when it comes to the timeframe the data refers to. I think there is a need of capturing and sharing also historical data. A relationship that existed in the past but is not active now. 

- This way we do not unknowingly rely on outdated or inaccurate data
- Traceability becomes more complete, increasing accountability in the supply chain

@wonderchook should I open this discussion as a separate issue?

**Comment by shuyag** [comment]
I wonder if the period of validity is a set time (frequency of reporting) or if it's set by the exchanger of data (i.e if it's a relationship, you know that the contract is 3 years long, or if it's that a location is within a location it's a longer time period)

**Comment by shuyag** [comment]
This seems like a field that maybe is either optional or is required, with the ability to have "unknown" or allow a descriptive time frame rather than expecting a specific date.

**Comment by wonderchook** [comment]
I think that time should not be a required field, but should be optional. There are different uses of data and time might not be required/important to every use case
